### ISSUE TITLE
lsp: relative import map values don't parse properly on windows

### ISSUE THREAD

**Issue by kecrily** [denoland/deno#11545]
**Describe the bug**

When using import_map.json and the module imports only one file. An `import-prefix-missing` error will be thrown.

**To Reproduce**

```ts
import { Command } from "cliffy";
```

import_map

```json
{
  "imports": {
    "cliffy": "https://deno.land/x/cliffy@v0.19.4/mod.ts"
  }
}
```

**Expected behavior**

Everything is fine

**Screenshots**

![图片](https://user-images.githubusercontent.com/45708948/127364908-f8576656-ac40-443a-85f7-8646ac409b29.png)

**Versions**

```shell
$ code -v
1.58.2
c3f126316369cd610563c75b1b1725e0679adfb3
x64
```

```shell
$ deno --version
deno 1.12.2 (release, x86_64-pc-windows-msvc)
v8 9.2.230.14
typescript 4.3.5
```

extension: v3.7.0

**Comment by amithm7** [comment]
the `deno.importMap` setting doesn't seem to get updated in _Deno Language Server Status_ until I had disabled and enabled the extension again.

**Comment by chrisdavies** [comment]
For the record, this is happening for me in Neovim too, so if there is a workaround, it'd be nice for it to work across all editors.

**Comment by bartlomieju** [comment]
> For the record, this is happening for me in Neovim too, so if there is a workaround, it'd be nice for it to work across all editors.
> 
> Edit: It seems like this is something that could be added to the `deno.json` file. Maybe, an "importMap" setting, or even moving the import maps into the `deno.json` itself.

You can specify path to import map in the config file starting in v1.20 https://deno.com/blog/v1.20#import-map-specified-in-the-configuration-file

**Comment by chrisdavies** [comment]
You're right! I was just editing my comment to mention that when I saw your reply. Thanks!
